src/test/java/uk/gov/digital/ho/pttg/ArchiveServiceTest.java

```diff
@@ -205,6 +208,44 @@ public void archiveResult_multipleArchivesFoundForDate_errorLogged() {
                 .isEqualTo(new ObjectAppendingMarker("event_id", PTTG_AUDIT_ARCHIVE_FAILURE));
     }
 
+    @Test
+    public void handleArchiveRequest_callsCollaboratorsWithParameters() {
+        LocalDate lastArchiveDate = LocalDate.now();
+        LocalDateTime lastArchiveDateTime = LocalDate.now().atTime(23, 59, 59, 999999999);
+        String nino = "any-nino";
+        List<String> eventIds = asList("any-corr-id-1", "any-corr-id-2");
+        LocalDate resultDate = LocalDate.now().minusDays(1);
+        String result = "PASS";
+
+        when(mockRepository.countNinosAfterDate(lastArchiveDateTime, nino)).thenReturn(0L);
+        when(mockRepository.findArchivedResults(resultDate.atStartOfDay(), resultDate.plusDays(1).atStartOfDay())).thenReturn(new ArrayList<>());
```

Comment:
Would Collections.emptyList() be more expressive than new ArrayList<>()? Collections.emptyList() also has the benefit that it is immutable - it's a good way of ensuring that we aren't reliant on the mutability of the returned List when we can't be sure of it.